### PR TITLE
LIME-102: Fixed log in

### LINKED ISSUE #102: ENHANCEMENT: Log in button should be in the normal state with empty mandatory fields and after negative validation
Log in button should be in the normal state with empty mandatory fields and after negative validation of values in the fields. Error message is displayed under empty field/field with invalid value after user clicks on Log in button. Logic of front-end validation should be aligned with the way it was implemented on Sign up page.

### PR DESCRIPTION
**Pull request by SergiiTernytskyi**
Fixed log in

### CODE REVIEW

**Comment by SergiiTernytskyi** [comment]
login
signin

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]
image
It looks like we need to update the style to fit the design
Also, I don't understand why in one place we have social buttons, but in another, we don't have

**Comment by SergiiTernytskyi** [comment]
login-2
signup-2

File: backend/src/common/database/base-database.package.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ class BaseDatabase implements Database {
 
     private get initialConfig(): TKnex.Config {
         const sslConfig =
-            this.appConfig.ENV.APP.ENVIRONMENT === AppEnvironment.LOCAL
+            this.appConfig.ENV.APP.ENVIRONMENT === AppEnvironment.DEVELOPMENT
```
Comment (anton-otroshchenko) [inline backend/src/common/database/base-database.package.ts]: why did you change this?

Comment (SergiiTernytskyi) [reply backend/src/common/database/base-database.package.ts]: Sorry I forgot to change it back

**Review by dimapopovych — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**